src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -175,11 +175,16 @@ private void StartBitcoinCoreRunner()
 
         private void StartStratisRunner()
         {
-            var cancellationToken = new CancellationTokenSource(TimeSpan.FromMinutes(1)).Token;
-            TestHelper.WaitLoop(() => this.runner.FullNode != null, cancellationToken: cancellationToken);
+            var timeToNodeInit = TimeSpan.FromMinutes(1);
+            var timeToNodeStart = TimeSpan.FromMinutes(1);
 
-            cancellationToken = new CancellationTokenSource(TimeSpan.FromMinutes(1)).Token;
-            TestHelper.WaitLoop(() => this.runner.FullNode.State == FullNodeState.Started, cancellationToken: cancellationToken);
+            TestHelper.WaitLoop(() => this.runner.FullNode != null, 
+                cancellationToken: new CancellationTokenSource(timeToNodeInit).Token,
+                failureReason: $"Failed to assingn instance of FullNode within {timeToNodeInit}");
```

Comment:
Typo

Reply:
I accept your comment and will make changes